### PR TITLE
Update LiveCELL inference scripts

### PR DESCRIPTION
**Pull request by anwai98**

### CODE REVIEW

File: finetuning/livecell/evaluation/evaluate_amg.py [hunk 1]
```diff
@@ -21,28 +24,33 @@ def run_amg(name, model_type, checkpoint):
...
+
+    save_path = os.path.join(experiment_folder, "results", "amg.csv")
     res = run_evaluation(gt_paths, pred_paths, save_path=save_path)
     print(res)
 
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-n", "--name", required=True)
+    parser.add_argument("-n", "--name", default=None)
```
Comment (constantinpape) [inline finetuning/livecell/evaluation/evaluate_amg.py:41]: I would avoid creating two different arguments `name` and `experiment_folder`. If you think it's better to pass the experiment_folder then let's remove the `name` argument and make the experiment folder mandatory.

File: finetuning/livecell/evaluation/evaluate_instance_segmentation.py [hunk 2]
```diff
@@ -21,28 +24,35 @@ def run_instance_segmentation_with_decoder(name, model_type, checkpoint):
...
+
+    save_path = os.path.join(experiment_folder, "results", "instance_segmentation_with_decoder.csv")
     res = run_evaluation(gt_paths, pred_paths, save_path=save_path)
     print(res)
 
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument("-n", "--name", required=True)
+    parser.add_argument("-n", "--name", default=None)
```
Comment (constantinpape) [inline finetuning/livecell/evaluation/evaluate_instance_segmentation.py:41]: See comment above.

File: finetuning/livecell/evaluation/iterative_prompting.py [hunk 3]
```diff
@@ -61,12 +61,13 @@ def evaluate_interactive_prompting(prediction_root, start_with_box_prompt, name)
 def main():
     parser = argparse.ArgumentParser()
 
-    parser.add_argument("-n", "--name", required=True)
+    parser.add_argument("-n", "--name", default=None)
```
Comment (constantinpape) [inline finetuning/livecell/evaluation/iterative_prompting.py:64]: See comment above.

File: finetuning/livecell/evaluation/evaluate_amg.py (hunk 1 above)
Comment (anwai98) [reply finetuning/livecell/evaluation/evaluate_amg.py]: Done! Thanks for the suggestion!

**Review by constantinpape — APPROVED** [review]

**Pull request merged**